test/unit/insights.test.js

```diff
@@ -28,14 +28,14 @@ describe('Application Insights', () => {
 
   test('does not setup application insights if no connection string present', () => {
     const appInsights = require('../../app/insights')
-    process.env.APPINSIGHTS_CONNECTIONSTRING = undefined
+    process.env.APPLICATIONINSIGHTS_CONNECTION_STRING = undefined
```

Comment:
This is aesthetic only as the variable name set and used in our service works without the change. I don't feel it was needed and only added to the scope of the ticket.